src/main/webapp/database-tables.js

```diff
@@ -22,33 +28,34 @@ function showDatabase() {
       for (tableIndex in data) {
         const tableData = data[tableIndex];
         const name = tableData.name;
-
         const isEmpty = tableData.isEmpty;
-        if (isEmpty) {
-          createIsEmptyElement(dataArea);
-          continue;
-        }
 
         const colSchemas = tableData.columnSchemas;
+        var finalDataTable = tableData;
+    
+        for (var i =0; i<colSchemas.length; i++) {
+            if (colSchemas[i].columnName.endsWith('Millis')){
+                var finalDataTable = dataConversionMillis(i,finalDataTable);
+            } else if (colSchemas[i].columnName == "Genre") {
+                var finalDataTable = dataConversionEnum(i,finalDataTable);
```

Comment:
Rename dataConversionEnum to dataConversionGenreEnum. 

Enum conversion without know what the enum is hard to generalize, we will probably need a map from the column name to the enums. For now, I think this looks great! Just try to specify that this works for Genre and will need to extend to other enums.

Reply:
Thank you, this has been changed!